### ISSUE TITLE
Inner join not implemented

### ISSUE THREAD

**Issue by trstovall** [rapidsai/dask-cudf#82]
Repro:
```
>>> import cudf, dask_cudf
>>> x = cudf.DataFrame({'a': [1, 2]})
>>> x
<cudf.DataFrame ncols=1 nrows=2 >
>>> y = dask_cudf.from_cudf(x, npartitions=1)
>>> y
<dask_cudf.DataFrame | 1 tasks | 1 npartitions>
>>> y.merge(y, on=('a',), how='inner')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/conda/envs/rapids/lib/python3.7/site-packages/dask_cudf-0+unknown-py3.7.egg/dask_cudf/core.py", line 255, in merge
    assert how == "left", "left join is impelemented"
AssertionError: left join is impelemented
```

**Comment by trstovall** [comment]
I believe this would be fixed along with #79.

**Comment by mrocklin** [comment]
Yup, as a warning, I wouldn't expect that effort any sooner than a month
from now
